scripts/e2e.sh

```diff
@@ -59,14 +72,15 @@ export SIPPY_ENDPOINT="127.0.0.1"
   --enable-write-endpoints \
   --log-level debug \
   --views config/views.yaml \
-  --google-service-account-credential-file $GCS_SA_JSON_PATH > e2e.log 2>&1
+  --google-service-account-credential-file $GCS_SA_JSON_PATH \
+  --redis-url="$REDIS_URL" > e2e.log 2>&1
 )&
 # store the child process for cleanup
 CHILD_PID=$!
 
-# Give it time to start up
+# Give it time to start up, and fill the redis cache
 echo "Waiting for sippy API to start on port $SIPPY_API_PORT, see e2e.log for output..."
-TIMEOUT=120
+TIMEOUT=600
```

Comment:
This timeout increase doesn't actually make the test any longer. The time is saved when actually running the tests. It is a longer startup time, but overall is very similar in length.